packages/cypress/tests/cypress/e2e/options.cy.ts

```diff
@@ -1,4 +1,4 @@
-it('Options / delay', { env: { delay: 1200 } }, () => {
+it('Options / delay', { env: { delay: 5000 } }, () => {
```

Comment:
Can the delay be made less (maybe by speeding up the gif frames, for example)? I'm assuming this will delay test runs by 5 seconds.

Reply:
It will only delay the Chromatic snapshot, not the test suite.

Reply:
I cut it back a bit!